routes/routeStack.js

```diff
@@ -1,13 +1,11 @@
 import { createStackNavigator, TransitionPresets } from "react-navigation-stack";
```

Comment:
<img width="953" alt="Screen Shot 2020-04-01 at 4 24 14 PM" src="https://user-images.githubusercontent.com/35850933/78183029-3b960e80-7435-11ea-8032-baf5e03860bb.png">

Reply:
Will see to it!